src/main/java/org/jabref/gui/DuplicateResolverDialog.java

```diff
@@ -39,108 +33,91 @@
         BREAK
     }
 
-    JButton helpButton = new HelpAction(Localization.lang("Help"), HelpFile.FIND_DUPLICATES).getHelpButton();
-    private final JButton cancel = new JButton(Localization.lang("Cancel"));
-    private final JButton merge = new JButton(Localization.lang("Keep merged entry only"));
+    ActionFactory factory = new ActionFactory(Globals.prefs.getKeyBindingRepository());
```

Comment:
I think all (most?) of these fields can be converted to local variables in `init`